_docs_integrate/version-5-migration-guide.md

```diff
@@ -21,85 +21,98 @@ required_by:
-| **creationContent** | `any`                                                                                             | The `creationContent` regarding this Relationship.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | **new**                                                                                        |
-| **auditLog**        | `RelationshipAuditLogEntry[]`                                                                     | The history of changes made to this Relationship.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | **new**                                                                                        |
+- The [Synchronize updates of Backbone]({% link _docs_use-cases/use-case-transport-synchronize-updates-of-backbone.md %}) use case no longer returns any content. Previously, new Messages or changes to Relationships were shown in the response after the use case was executed. As the [synchronization with the Backbone no longer returns anything]({% link _docs_integrate/version-5-migration-guide.md %}#synchronization-with-backbone-returns-no-content-anymore), the corresponding [events]({% link _docs_integrate/connector-events.md %}) must be listened to instead in order to be informed about new Messages or changes to Relationships.
+- Stricter [validation of Requests]({% link _docs_integrate/version-5-migration-guide.md %}#validation-of-requests) has been added. Therefore, changes will probably have to be made to existing Request flows. Both [sending Requests]({% link _docs_integrate/version-5-migration-guide.md %}#sending-of-requests) and [responding to Requests]({% link _docs_integrate/version-5-migration-guide.md %}#responding-to-requests) are affected.
 
-### RelationshipAuditLogEntry
+### Changes to Error Codes, Use Cases and Events
```

Comment:
the integrator typically doesn't think in Use Cases instead they think about routes. Maybe we can find a better wording here.